test/v2/services/MessagesXmlParsingServiceSpec.scala

```diff
@@ -89,5 +90,17 @@ class MessagesXmlParsingServiceSpec
         _ mustBe Left(ExtractionError.MessageTypeNotFound("HolderOfTheTransitProcedure"))
       }
     }
+
+    "if the input is malformed, return EctractionError.MalformedInput" in {
```

Comment:
EctractionError -> ExtractionError :')

Reply:
I'll fix that... got the arrivals stuff to shoe in too.